http-client/src/settings.rs

```diff
@@ -0,0 +1,82 @@
+use serde::Deserialize;
```

Comment:
I took this package from [here](https://github.com/p2p-org/token-service/blob/main/http-client/src/lib.rs)